vehicles.ini

```
@@ -5,19 +5,19 @@ audir18,Audi R18,6,4200
 audir8gt3,Audi R8 LMS,6,7000
 audirs3lms,Audi RS 3 LMS,6,6600
 bmwm8gte,BMW M8 GTE,6,6800
-bmwm4gt3,BMW M4 GT3,6,6700
 bmwm4gt4,BMW M4 GT4,5,7100
+bmwm4gt3,BMW M4 GT3,6,6650
 bmwz4gt3,BMW Z4 GT3,6,8400
 cadillacctsvr,Cadillac CTS-V Racecar,6,7400
 c6r,Chevrolet Corvette C6.R GT1,6,5700
 c7vettedp,Chevrolet Corvette C7 Daytona Prototype,6,6800
-c8rvettegte,Chevrolet Corvette C8.R GTE,6,6500
+c8rvettegte,Chevrolet Corvette C8.R,6,6800
```

Alias please revert or provide reason
RPM is ok
```suggestion
c8rvettegte,Chevrolet Corvette C8.R GTE,6,6800
```